src/dotnet/APIView/APIViewWeb/Client/src/review.ts

```diff
@@ -1,25 +1,109 @@
 ﻿$(() => {  
-  const SEL_DOC_CLASS = ".documentation";
-  const SHOW_DOC_CHECK_COMPONENT = "#show-documentation-component";
-  const SHOW_DOC_CHECKBOX = ".show-doc-checkbox";
-  const SHOW_DOC_HREF = ".show-document";
-  const SHOW_DIFFONLY_CHECKBOX = ".show-diffonly-checkbox";
-  const SHOW_DIFFONLY_HREF = ".show-diffonly";
+    const SEL_DOC_CLASS = ".documentation";
+    const SHOW_DOC_CHECK_COMPONENT = "#show-documentation-component";
+    const SHOW_DOC_CHECKBOX = ".show-doc-checkbox";
+    const SHOW_DOC_HREF = ".show-document";
+    const SHOW_DIFFONLY_CHECKBOX = ".show-diffonly-checkbox";
+    const SHOW_DIFFONLY_HREF = ".show-diffonly";
 
-  hideCheckboxIfNoDocs();
+    hideCheckboxIfNoDocs();
 
-  function hideCheckboxIfNoDocs() {
-      if ($(SEL_DOC_CLASS).length == 0) {
-          $(SHOW_DOC_CHECK_COMPONENT).hide();
-      }
-  }
+    function hideCheckboxIfNoDocs() {
+        if ($(SEL_DOC_CLASS).length == 0) {
+            $(SHOW_DOC_CHECK_COMPONENT).hide();
+        }
+    }
 
-  $(SHOW_DOC_CHECKBOX).on("click", e => {
-    $(SHOW_DOC_HREF)[0].click();
-  });
+    $(SHOW_DOC_CHECKBOX).on("click", e => {
+        $(SHOW_DOC_HREF)[0].click();
+    });
 
-  $(SHOW_DIFFONLY_CHECKBOX).on("click", e => {
-    $(SHOW_DIFFONLY_HREF)[0].click();
-  });
+    $(SHOW_DIFFONLY_CHECKBOX).on("click", e => {
+        $(SHOW_DIFFONLY_HREF)[0].click();
+    });
 
+    // Add a mode for keyboard navigation between diff blocks.  This is an
```

Comment:
Github has this always on. Should we do it too?

A somewhat related idea for the future: next/previous diff toolbar buttons.

Reply:
I worry that it's kind of jarring if you're not expecting it.  Especially because we're not doing anything intelligent like calculating the the next diff relative to what's on the screen.  If you've reviewed most of the API and then hit `j` randomly, it'll jump you back to the first diff in the document.  I'd suggest leaving it disabled by default until it's less hacky, but don't feel strongly either way.